### DISCUSSION TITLE
Get meal plan for a date

### DISCUSSION THREAD

**Discussion by JoshuaWingert** [mealie-recipes/mealie#2130] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Looking for a way to call the api for a given date it would return the full meal plan for the date.  Currently with the /api/meal-plans/today it just shows the main course, does not show sides.  There was mention of an iCal but the issue has staled. I was looking to integrate into HA and show the next 5 days of dinners. 

**(Optional): Suggest A Solution**
Create an endpoint of /api/meal-plans/date/yyyy-mm-dd that would return the full meal plan for that date.

**Comment by hay-kot** [comment]
https://demo.mealie.io/docs#/Groups%3A%20Mealplans/get_all_api_groups_mealplans_get

Should be what you're looking for

**Reply by JoshuaWingert** [reply]
Thanks.  I will look into it, it looks like that is on the beta which I am not running.  I'll run a new stack tonight and see if that resolves the issue.
